sys/net/include/etx_beaconing.h

```diff
@@ -26,43 +26,45 @@
+ */
 #define ETX_BUF_SIZE   (32)
-
 #define ETX_RCV_QUEUE_SIZE     (128)
 
-/*
- * Default 40, should be enough to get all messages for neighbors.
- * In my tests, the maximum count of neighbors was around 30-something
+/**  
+ * @brief defining the number of neighbours 
  */
-#if ENABLE_DEBUG
-#define ETX_MAX_CANDIDATE_NEIGHBORS (15) //Stacksizes are huge in debug mode, so memory is rare
+#if ENABLE_DEBUG                                     
+#define ETX_MAX_CANDIDATE_NEIGHBORS (15)             /**< Stacksizes are huge in debug mode, so memory is rare so we fix it at 15 */
 #else
-#define ETX_MAX_CANDIDATE_NEIGHBORS (40)
+#define ETX_MAX_CANDIDATE_NEIGHBORS (40)             /**< Default 40, should be enough to get all messages for neighbors. */
 #endif
-//ETX Interval parameters
 #define MS  (1000)
 
-/*
- * ETX_INTERVAL
- *
+/**
+ * @{
+ * @name ETX Interval parameters
  * Given in ms, the default is 1 second.
  * Should be divisible through 2 (For ETX_DEF_JIT_CORRECT)
  * and 5 (For ETX_MAX_JITTER) unless those values are adjusted too.
  */
 #define ETX_INTERVAL        (1000)
-#define ETX_WINDOW          (10)                    //10 is the default value
-#define ETX_BEST_CANDIDATES (15)                    //Sent only 15 candidates in a beaconing packet
-#define ETX_TUPLE_SIZE      (2)                     //1 Byte for Addr, 1 Byte for packets rec.
-#define ETX_PKT_REC_OFFSET  (ETX_TUPLE_SIZE - 1)    //Offset in a tuple of (addr,pkt_rec), will always be the last byte
-#define ETX_IPV6_LAST_BYTE  (15)                    //The last byte for an ipv6 address
-#define ETX_MAX_JITTER      (ETX_INTERVAL / 5)      //The default value is 20% of ETX_INTERVAL
-#define ETX_JITTER_MOD      (ETX_MAX_JITTER + 1)    //The modulo value for jitter computation
-#define ETX_DEF_JIT_CORRECT (ETX_MAX_JITTER / 2)    //Default Jitter correction value (normally ETX_MAX_JITTER / 2)
-#define ETX_CLOCK_ADJUST    (52500)                 //Adjustment for clockthread computations to stay close/near ETX_INTERVAL
+#define ETX_WINDOW          (10)                    /**< 10 is the default value */
+#define ETX_BEST_CANDIDATES (15)                    /**< Sent only 15 candidates in a beaconing packet */
+#define ETX_TUPLE_SIZE      (2)                     /**< 1 Byte for Addr, 1 Byte for packets rec. */
+#define ETX_PKT_REC_OFFSET  (ETX_TUPLE_SIZE - 1)    /**< Offset in a tuple of (addr,pkt_rec), will always be the last byte */
+#define ETX_IPV6_LAST_BYTE  (15)                    /**< The last byte for an ipv6 address */
+#define ETX_MAX_JITTER      (ETX_INTERVAL / 5)      /**< The default value is 20% of ETX_INTERVAL */
+#define ETX_JITTER_MOD      (ETX_MAX_JITTER + 1)    /**< The modulo value for jitter computation */
+#define ETX_DEF_JIT_CORRECT (ETX_MAX_JITTER / 2)    /**< Default Jitter correction value (normally ETX_MAX_JITTER / 2) */
+#define ETX_CLOCK_ADJUST    (52500)                 /**< Adjustment for clockthread computations to stay close/near ETX_INTERVAL */
```

Comment:
some of the above comments are a bit long, maybe it is better for readability to put all of them on separate lines.